GANDLF/entrypoints/construct_csv.py

```diff
@@ -0,0 +1,181 @@
+    "--channels-id _t1.nii.gz,_t2.nii.gz. May be a YAML file with `channels` list of suffixes",
+    type=str,
+)
+@click.option(
+    "--label-id",
+    "-l",
+    type=str,
+    help="Label/mask identifier string to check for in all files in 'input_dir'; for example: "
+    "--label-id _seg.nii.gz. Param value is ignored in `label` is defined in channels YAML file",
+)
+@click.option(
+    "--output-file",
+    "-o",
+    required=True,
+    type=click.Path(file_okay=True, dir_okay=False),
+    help="Output CSV file",
+)
+@click.option(
+    "--relativize-paths",
+    "-r",
+    is_flag=True,
+    help="If True, paths in the output data CSV will always be relative to the location"
+    " of the output data CSV itself.",
+)
+@append_copyright_to_help
+def new_way(
+    input_dir: str,
+    channels_id: str,
+    label_id: Optional[str],
+    output_file: str,
+    relativize_paths: bool,
+):
+    """Generate training/inference CSV from data directory."""
+    _construct_csv(
+        input_dir=input_dir,
+        channels_id=channels_id,
+        label_id=label_id,
+        output_file=output_file,
+        relativize_paths_to_output=relativize_paths,
+    )
+
+
+@deprecated(
+    "This is a deprecated way of running GanDLF. Please, use `gandlf construct-csv` cli command "
+    + "instead of `gandlf_constructCSV`. Note that in new CLI tool some params were renamed:\n"
+    + "  --inputDir to --input-dir\n"
+    + "  --channelsID to --channels-id\n"
+    + "  --labelID to --label-id\n"
+    + "  --outputFile to --output-file\n"
+    + "  --relativizePaths to --relativize-paths and converted to flag, i.e. no value required\n"
+    + "`gandlf_constructCSV` script would be deprecated soon."
+)
+def old_way():
+    logger_setup()
+    parser = argparse.ArgumentParser(
+        prog="GANDLF_ConstructCSV",
+        formatter_class=argparse.RawTextHelpFormatter,
+        description="Generate training/inference CSV from data directory.\n\n"
+        + copyrightMessage,
+    )
+    parser.add_argument(
+        "-i",
+        "--inputDir",
+        metavar="",
+        type=str,
+        help="Input data directory which contains images in specified format",
+    )
+    parser.add_argument(
+        "-c",
+        "--channelsID",
+        metavar="",
+        type=str,
+        help="Channels/modalities identifier string to check for in all files in 'input_dir'; for example: --channelsID _t1.nii.gz,_t2.nii.gz",
+    )
+    parser.add_argument(
+        "-l",
+        "--labelID",
+        default=None,
+        type=str,
+        help="Label/mask identifier string to check for in all files in 'input_dir'; for example: --labelID _seg.nii.gz",
+    )
+    parser.add_argument(
+        "-o", "--outputFile", metavar="", type=str, help="Output CSV file"
+    )
+    parser.add_argument(
+        "-r",
+        "--relativizePaths",
+        metavar="",
+        type=ast.literal_eval,
```

Comment:
should we also go there with `store_true` action instead of literal eval type? This would remove the need to write `True` or `False` as arg, but it is breaking change for previous api. If you think we should maintain compatibility this comment is irrelevant

Reply:
We need to maintain compatibility with current API.